app/models/customer_return_decorator.rb

```diff
@@ -0,0 +1,83 @@
+module SolidusShipwire::CustomerReturn
+  prepend SolidusShipwire::Proxy
+
+  def self.prepended(base)
+    base.after_save :process_shipwire_return!, if: :create_on_shipwire?
+  end
+
+  def to_shipwire
+    {
+      externalId: order.number,
+      originalOrder: {
+        id: order.shipwire_id.to_s
+      },
+      items:
+        shipwire_line_items.map { |s, q| { sku: s, quantity: q } },
+      options: {
+        generatePrepaidLabel: generate_prepaid_label,
+        emailCustomer: email_customer,
+        warehouseId: warehouse_id,
+        warehouseExternalId: warehouse_external_id,
+        warehouseRegion: warehouse_region
+      }
+    }
+  end
+
+  private
+
+  def create_on_shipwire?
+    shipwire_order.present?
+  end
+
+  def process_shipwire_return!
+  create_on_shipwire
+  rescue SolidusShipwire::ResponseException => e
+    if e.response.validation_errors
```

Comment:
If the condition is false I do nothing. This is the case of a rare Timeout exception. All the other exceptions have validation errors and I forward them to the object. Timeout does not have any validation errors attached. How should I rescue from it?